### DISCUSSION TITLE
how to use .\"Office Tool Plus.Console"

### DISCUSSION THREAD

**Discussion by hieuck** [YerongAI/Office-Tool#681] [Q&A]
tôi cần tìm tài liệu để sử dụng Office Tool Plus.Console
I need to find documentation to use Office Tool Plus.Console
我需要找到使用 Office Tool Plus.Console 的文档

0
第一的

**Comment by hieuck** [comment]
https://help.coolhub.top/others/commands.html#office-tool-plus-console-helper

tôi đã tìm thấy. đóng chủ đề
I've found. close topic

**Comment by hieuck** [comment]
làm thế nào để đặt tuỳ chọn tắt cập nhật trong console?
how to set the option to disable updates in console?
如何设置选项以禁用控制台中的更新？

**Comment by hieuck** [comment]
làm thế nào để "Close" khi cài đặt office hoàn tất?
How do I 'Close' when Office installation is complete?

**Comment by MineTurtlee** [comment]
còn chưa xong á hả?
